fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraReindexIT.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Licensed to DuraSpace under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * DuraSpace licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except in
+ * compliance with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.fcrepo.integration.http.api;
+
+import org.apache.commons.io.FileUtils;
+import org.apache.http.HttpStatus;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.test.context.TestExecutionListeners;
+
+import java.nio.file.Path;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * @author dbernstein
+ * @since 12/01/20
+ */
+@TestExecutionListeners(
+        listeners = {LinuxTestIsolationExecutionListener.class},
+        mergeMode = TestExecutionListeners.MergeMode.MERGE_WITH_DEFAULTS)
+public class FedoraReindexIT extends AbstractResourceIT {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(FedoraReindexIT.class);
+
+    @Test
+    public void testSideLoading() throws Exception {
```

Comment:
This is a happy path test. Other test ideas:

1. What happens if you attempt to reindex an object that does not exist?
2. What happens if you attempt to reindex an archival group part rather than the ag? For example, `info:fedora/ag/child` rather than `info:fedora/ag`
3. What happens if the object exists but is invalid?